src/bundle/Resources/views/admin/search/search_form.html.twig

```diff
@@ -10,6 +10,100 @@
                 <use xlink:href="{{ asset('bundles/ezplatformadminui/img/ez-icons.svg') }}#search"></use>
             </svg>
             {{ 'search.perform'|trans|desc('Search') }}</button>
+        <button type="submit" class="btn btn-dark ez-btn--filter ml-4">
+            <svg class="ez-icon ez-icon--small ez-icon--light ez-icon-filters">
```

Comment:
Do you need this: `ez-icon-filters` - class name?

Reply:
For consistency, all icons have own class name

Reply:
Of course, but do these classes affect the final result?

Reply:
No, we don't style by this class. It's for end-developer who would like to change the styling of the icon.

Reply:
IMO, we should drop the class name as it's not used anywhere.